src/components/views/settings/devices/SecurityRecommendations.tsx

```diff
@@ -29,19 +29,26 @@ import {
 
 interface Props {
     devices: DevicesDictionary;
+    currentDeviceId: DeviceWithVerification['device_id'];
     goToFilteredList: (filter: DeviceSecurityVariation) => void;
 }
 
 const SecurityRecommendations: React.FC<Props> = ({
     devices,
+    currentDeviceId,
     goToFilteredList,
 }) => {
     const devicesArray = Object.values<DeviceWithVerification>(devices);
 
     const unverifiedDevicesCount = filterDevicesBySecurityRecommendation(
         devicesArray,
         [DeviceSecurityVariation.Unverified],
-    ).length;
+    )
+        // filter out the current device
+        // as unverfied warning and actions
+        // will be shown in current session section
```

Comment:
With the short line length this reads a bit like a poem :smile:

Reply:
accidental haiku :)